src/main/java/com/bnkrll/controller/SessionController.java

```diff
@@ -35,4 +35,18 @@ public List<Session> getPreviousSessions(
                     int last) {
         return sessionRepository.getLastSessions(last);
     }
+
+    @PatchMapping
+    public ResponseEntity<Session> updateSession(@RequestBody Session session){
+        List<Session> lastSession = sessionRepository.getLastSessions(1);
+        if(lastSession.isEmpty()){
```

Comment:
remember to format your files before committing

Reply:
Done !